dbms/build.gradle

```diff
@@ -50,6 +50,8 @@ dependencies {
 
     // workaround to fix cottontail todo remove: when grpc-all is same in cottontail and for plugin
     implementation group: "io.grpc", name: "grpc-all", version: cottontaildb_grpc_version
+    // workaround so org.polypheny.db.docker gets the right protobuf during testing
```

Comment:
Why during testing? Protobuf needs to be imported for this module, since Docker lives here...

Reply:
Docker is in the core.  Polypheny compiles and starts fine without this line, but for the integration tests, a stacktrace like the following appears when talking to Docker:
```
  Exception in thread "Thread-25" java.lang.UnsupportedOperationException: This is supposed to be overridden by subclasses.
        at com.google.protobuf.GeneratedMessageV3.getUnknownFields(GeneratedMessageV3.java:302)
        at org.polypheny.db.docker.VersionRequest.getSerializedSize(VersionRequest.java:82)
        at com.google.protobuf.CodedOutputStream.computeMessageSizeNoTag(CodedOutputStream.java:877)
        at com.google.protobuf.CodedOutputStream.computeMessageSize(CodedOutputStream.java:661)
        at org.polypheny.db.docker.Request.getSerializedSize(Request.java:648)
        at com.google.protobuf.AbstractMessageLite.writeDelimitedTo(AbstractMessageLite.java:89)
        at org.polypheny.db.docker.PolyphenyDockerClient.executeRequest(PolyphenyDockerClient.java:75)
        at org.polypheny.db.docker.PolyphenyDockerClient.<init>(PolyphenyDockerClient.java:38)
        at org.polypheny.db.docker.DockerInstance.connectToDocker(DockerInstance.java:106)
        at org.polypheny.db.docker.DockerInstance.checkConnection(DockerInstance.java:144)
        at org.polypheny.db.docker.DockerInstance.<init>(DockerInstance.java:96)
        at org.polypheny.db.docker.DockerManager.addDockerInstance(DockerManager.java:100)
        at org.polypheny.db.docker.DockerSetupHelper.lambda$newDockerInstance$0(DockerSetupHelper.java:75)
        at org.polypheny.db.docker.PolyphenyHandshakeClient.doHandshake(PolyphenyHandshakeClient.java:249)
...
```